src/Lykke.Quintessence.SignService.Domain/Services/IWalletService.cs

```diff
@@ -0,0 +1,9 @@
+using System.Threading.Tasks;
+
+namespace Lykke.Quintessence.Domain.Services
+{
+    public interface IWalletService
+    {
+        Task<(string Address, string PrivateKey)> CreateWalletAsync();
```

Comment:
Contract contains also AddressContext. it may be required for some implementations.